### DISCUSSION TITLE
Can's use Chinese in Page name and directory

### DISCUSSION THREAD

**Discussion by quank123wip** [requarks/wiki#3522] [Error / Bug Report]
**Describe the bug**
When I'm trying to use Chinese in title and directory name, it just can't save or turn into a blank.

**To Reproduce**
Use the latest Docker Image, Enter a Chinese name.

**Expected behavior**
Can use Chinese in title and directory

**Host Info (please complete the following information):**
 - OS: Docker
 - Wiki.js version: 2.3.81
 - Database engine: PostgreSQL 11.8

**Comment by wlfcss** [comment]
This bug seems to be related to the Chinese input method，when you use ctrl+c & ctrl+v ,that doesn't happen.

**Comment by joshuaavalon** [comment]
I think this bug is not related Chinese input method. Other than Chinese, Japanese has the same problem. I guess it is related to the handling language input method that supports "converting" words.

**Comment by wlfcss** [comment]
> I think this bug is not related Chinese input method. Other than Chinese, Japanese has the same problem. I guess it is related to the handling language input method that supports "converting" words.

yes, your description is more correct.

**Comment by chisaato** [comment]
I found this problem only happens on chrome. On Firefox, it can accept Chinese .
In my opinion,I suggest using English in URL(directory) and create a same name doc for renaming the virtual folder. just use Chinese in the name of the doc. Not URL.
